CCS/Guadaloop_lib/guadaloop/lib/i2c/i2c_read_write.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * i2c_read_write.c
+ *
+ * Authors: Daniela Caballero, Ty Brinker
+ *
+ * */
+#include <stdint.h>
+#include "msp432e401y.h"
+#include <guadaloop/lib/i2c/i2c_read_write.h>
+/*
+ *
+ * @brief: functions and data structures to help with i2c communication
+ *
+ */
+
```

Comment:
add any comments that might help people in the future refactor this code if they wish to.